src/shogun/kernel/MaternKernel.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Gil Hoben
+ */
+
+#include <shogun/distance/EuclideanDistance.h>
+#include <shogun/kernel/MaternKernel.h>
+
+using namespace shogun;
+
+
+MaternKernel::MaternKernel(): ShiftInvariantKernel()
+{
+	register_params();
```

Comment:
I think with initializing parameters in the declaration and calling the base ctor in the other ones, we might not need these methods anymore, could just register directly here?